chips/lowrisc/src/virtual_otbn.rs

```diff
@@ -57,7 +57,7 @@ impl<'a> VirtualMuxAccel<'a> {
     pub fn load_data(
         &self,
         address: usize,
-        data: &'static mut [u8],
+        data: LeasableBuffer<'static, u8>,
```

Comment:
The intended use of `LeasableBuffer` is for the `LeasableBuffer` to be returned via a callback (or in the error case), and then `reset()` can be called to regain access to the entire buffer, but I don't see how `LeasableBuffer` is really providing much as used here.

Reply:
Interesting, previously I have been passing in a `LeasableBuffer` and then returning just the static buffer (https://github.com/tock/tock/blob/124e7ac9c97dbe9b4bc719a0b884ed827778f55e/kernel/src/hil/digest.rs#L76).

The idea being that the caller can specify a active slice using `LeasableBuffer` but then just wants the full buffer to be returned.

In this case it's used to provide an active slice, so the caller can specify a subset range of the buffer to be used

Reply:
I see. Okay sounds fine then